static/app/views/replays/details.tsx

```diff
@@ -113,17 +105,18 @@ function ReplayDetails() {
             </Panel>
           </ReplayLayout>
           <Layout.Side>
-            <UserActionsNavigator event={event} entry={breadcrumbEntry} />
+            <UserActionsNavigator
+              event={replay.getEvent()}
+              entry={replay.getEntryType(EntryType.BREADCRUMBS)}
```

Comment:
This change means that we're passing in ALL breadcrumbs again, no longer filtering the replay-transactions.

This is ok because inside `UserActionsNavigator` we need to do another filter anyway and ignore out all transactions among other things.